qa/L0_perf_analyzer_doc_links/test.sh

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LOG="`pwd`/log.txt"
```

Comment:
@debermudez  
1. Can you help me to figure out what is the case to use  `pwd` utility here?
I don't see any change path `cp` command in the script, there why I'm assume that it's redundant. 
2. `LOG` file I would recommend to declare with `*.log` extension to keep it consistent, as we have a patterns to collect log files like `**\*.log`.

Reply:
`pwd` was just to be consistent and use an absolute path over a relative path. I am happy to just use `.` instead if you think that is cleaner.

I will update the log file name right now.

Reply:
Regarding `pwd` it's up to you, in fact it will add few more lines on in CI log. 
I don't care if it working fine.